crates/sui-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -196,9 +215,45 @@ pub async fn serve_executor(
         snapshot_config: snapshot_config.unwrap_or_default(),
         graphql_connection_config,
         cancellation_token,
+        database,
+        tempdir: None,
     }
 }
 
+pub async fn prep_executor_cluster() -> ExecutorCluster {
+    let rng = StdRng::from_seed([12; 32]);
+    let data_ingestion_path = tempdir().unwrap();
+    let mut sim = Simulacrum::new_with_rng(rng);
+    sim.set_data_ingestion_path(data_ingestion_path.path().to_path_buf());
+
+    sim.create_checkpoint();
+    sim.create_checkpoint();
+    sim.create_checkpoint();
+    sim.advance_epoch(true);
+    sim.create_checkpoint();
+    sim.advance_clock(
+        std::time::SystemTime::now()
+            .duration_since(std::time::SystemTime::UNIX_EPOCH)
+            .unwrap(),
+    );
```

Comment:
should we advance clock for each checkpoint to better emulate what happens in prod

Reply:
this is specifically to get the clock to be close to "current time" for the health check test. there's probably some changes we could make to Simulacrum to make this a bit easier of course